bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -1913,6 +1924,10 @@ private abstract class AbstractImageProviderWrapper {
 
 	protected abstract Rectangle getBounds(int zoom);
 
+	protected boolean supportComplexZoom() {
```

Comment:
Can we find a more self-explaining name than "complex" for that kind of zoom? I think that one would need to look into the code to what's meant by "complex". Maybe we could even defer the logic of providing a proper `ZoomContext` to the provider, i.e., instead of an `supportComplexZoom()` we may have somelike like a `ZoomContext getFittingZoomContext(target, native)`.